### ISSUE TITLE
Property "link" links back to the wallabag item in starred feed

### ISSUE THREAD

**Issue by ddeimeke** [wallabag/wallabag#1947]
### Issue details

In the feed https://URL/USER/TOKEN/starred.xml (same for archive feed) the link-tag contains the URL to the article in Wallabag, which is the same as the guid-tag.

This should be the URL of the original page.

Example:
```
...
<title><![CDATA[The tool isn’t the problem… — The Startup]]></title>
<source url="https://wallabag.deimeke.net/view/228">wallabag</source>
<link>https://wallabag.deimeke.net/view/228</link>
<guid>https://wallabag.deimeke.net/view/228</guid>
<pubDate>Mon, 18 Apr 2016 10:48:48</pubDate>
...
```

Should be (in this case): https://medium.com/swlh/the-tool-isn-t-the-problem-5e19b6857e56#.dpbtzfvdt

### Environment

* wallabag version (or git revision) that exhibits the issue: 2.0.1
* How did you install wallabag? Via `git clone` or by downloading the package? git clone
* Last wallabag version that did not exhibit the issue (if applicable): 1.9.1 (never tested it with any other 2.0.x version)
* php version: 5.6.20
* OS: CentOS 7.2 (recent patchlevel)
* type of hosting (shared or dedicated): dedicated
* which storage system you choose at install (SQLite, MySQL/MariaDB or PostgreSQL): MariaDB

### Steps to reproduce/test case

Please provide necessary steps for reproduction of this issue, or better the
reduced test case (without any external dependencies, if possible).

**Comment by nicosomb** [comment]
Aaaand ... already fixed here #1929 ;-)

**Comment by ddeimeke** [comment]
Ok, great again :-)
